ghost/core/core/server/web/parent/middleware/queue-request.js

```diff
@@ -0,0 +1,76 @@
+const debug = require('@tryghost/debug')('middleware:queue-request');
+const metrics = require('@tryghost/metrics');
+const path = require('node:path');
+const expressQueue = require('express-queue');
+
+const DEFAULT_QUEUE_LIMIT = 100;
+
+module.exports = function queueRequest(
+    config,
+    queueFactory = expressQueue,
+    logMetric = metrics.metric.bind(metrics)
+) {
+    config = {
+        ...{
+            limit: DEFAULT_QUEUE_LIMIT
```

Comment:
`DEFAULT_QUEUE_LIMIT` is a confusing name to me because it's the active request limit. Could we change the naming to be `ACTIVE_REQUEST_LIMIT` or something similar?

Reply:
Yeh I fumbled the terminology here. How about `CONCURRENCY_LIMIT`?

Reply:
Works for me.

Reply:
👌 Updated in https://github.com/TryGhost/Ghost/pull/19887/commits/67856d3e804bd43257a9078a801070d769670808